### PR TITLE
grid_print.py and fizzbuzz.py submission

### PR DESCRIPTION
**Pull request by spacemollusk**
Seems to be working!

### CODE REVIEW

**Comment by spacemollusk** [comment]
Also added fizzbuzz.py.

File: students/jim/Lesson02/series.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        return(1)
+    else:
+        fib_2 = 1
+        fib_1 = 1
+        for i in range(2, n):
+            value = fib_1 + fib_2
+            print(float(value) / float(fib_2))
+            fib_1 = fib_2
+            fib_2 = value
+        return(value)
```
Comment (codedragon) [inline students/jim/Lesson02/series.py]: Neat python trick:
`fib_1, fib_2 = fib_2, value`

File: students/jim/Lesson02/series.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+def fibonacci_iterative(n):
+    if n < 3:
+        return(1)
+    else:
+        fib_2 = 1
+        fib_1 = 1
+        for i in range(2, n):
+            value = fib_1 + fib_2
+            print(float(value) / float(fib_2))
```
Comment (codedragon) [inline students/jim/Lesson02/series.py]: What is the print statement for? I wasn't aware you could learn anything from dividing a Fibonacci number by one of the addends.

File: students/jim/Lesson02/grid_print.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    for y in range(grid_size * cell_size):
+        if y % cell_size == 0:
+            print(horiz_line)
+        else:
+            print(vert_line)
+
+    print(horiz_line)
+
+
+print_grid(2, 2)
```
Comment (codedragon) [inline students/jim/Lesson02/grid_print.py]: Nice solution!

File: students/jim/Lesson02/series.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            fib_1 = fib_2
+            fib_2 = value
+        return(value)
+
+print(fibonacci_iterative(30))
+print(fibonacci_recursive(30))
```
Comment (codedragon) [inline students/jim/Lesson02/series.py]: I got curious, and timed the code. Turns out for Fibonacci, recursive and iterative take nearly the same amount of time. I commented out the print statement when I tested, as that is a big time drain.

**Pull request merged**